lib/matplotlib/contour.py

```diff
@@ -629,7 +629,7 @@ def _is_closed_polygon(X):
     presumably coordinates on a polygonal curve, in which case this function
     tests if that curve is closed.
     """
-    return np.all(X[0] == X[-1])
+    return np.all(np.isclose(X[0], X[-1]))
```

Comment:
Use np.allclose() directly.

https://numpy.org/doc/stable/reference/generated/numpy.allclose.html

Reply:
Yeah, that was just a lapse in my brain.